src/routes/NoteCreateRoute.js

```diff
@@ -33,9 +32,7 @@ const NoteCreateRoute = ({
 };
 
 NoteCreateRoute.propTypes = {
-  history: PropTypes.shape({
-    goBack: PropTypes.func.isRequired,
-  }).isRequired,
+  history: ReactRouterPropTypes.location.isRequired,
```

Comment:
should be `ReactRouterPropTypes.history.isRequired`, not `ReactRouterPropTypes.location.isRequired`